htmresearch/frameworks/nlp/classify_htm.py

```diff
@@ -84,6 +84,10 @@ def __init__(self,
       self.networkConfig["classifierRegionConfig"].get("regionName")]
 
 
+  def getKNN(self):
+    return self.classifierRegion._knn
```

Comment:
This doesn't do anything...

    (Pdb) self.classifierRegion
    <nupic.engine.Region object at 0x107cc5990>
    (Pdb) self.classifierRegion._knn
    *** AttributeError: 'Region' object has no attribute '_knn'

You may be looking for getSelf():

    (Pdb) self.classifierRegion.getSelf()
    <nupic.regions.KNNClassifierRegion.KNNClassifierRegion object at 0x107cc5390>